pkg/translator/azurelogs/resourcelogs_to_logs.go

```diff
@@ -141,7 +162,8 @@ func (r ResourceLogsUnmarshaler) UnmarshalLogs(buf []byte) (plog.Logs, error) {
 				// TODO @constanca-m This will be removed once the categories
 				// are properly mapped to the semantic conventions in
 				// category_logs.go
-				err = lr.Body().FromRaw(extractRawAttributes(log))
+				rawRecord := getRawRecord(i)
```

Comment:
Would there be any issues in the rest of this method if the returned raw record is empty or `nil`? (I'm wondering if we should check the returned value at all before continuing, or if it's harmless)